docs/source/knowledgebase/quasi_dags.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>I wonder if it's worth pivoting out of this example and saying something now about identification in general in causal inference. This ANCOVA is a strong example... to jump off and discuss how all the strategies above are identification strategies. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/517/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>